src/Microsoft.VisualStudio.Threading.Analyzers.Tests/VSTHRD002UseJtfRunAnalyzerTests.cs

```diff
@@ -54,13 +38,12 @@ class Test {
     }
 }
 ";
-            this.expect.Locations = new[] { new DiagnosticResultLocation("Test0.cs", 8, 14, 8, 18) };
-            this.VerifyCSharpDiagnostic(test, this.expect);
-            this.VerifyCSharpFix(test, withFix);
+            var expected = Verify.Diagnostic().WithSpan(8, 14, 8, 18);
+            await Verify.VerifyCodeFixAsync(test, expected, withFix);
```

Comment:
:memo: Something to be aware of - `VerifyCodeFixAsync` in the form seen here is equivalent to the following:

```
await Verify.VerifyAnalyzerAsync(test, expected); // Verify input has the right diagnostics
await Verify.VerifyAnalyzerAsync(withFix); // Verify the output has no diagnostics
[no direct representation] // Verify the code fix applied to the input produces the output
```